### PR TITLE
Add document full text search function (elastic search).

### PR DESCRIPTION
**Pull request by maliming**
We should design a search page, which can also be used for support forums.

This is a very simple search results page I made.

image

image

### CODE REVIEW

**Comment by maliming** [comment]
hi @armgnunlu 
Can you help design a search results page? We can use it in docs and qa modules.

**Comment by hikalkan** [comment]
I moved this to the next milestone to review it deeper and work on the UI design.

File: modules/docs/app/VoloDocs.Web/appsettings.json [hunk 1]
```diff
@@ -1,4 +1,8 @@
 ﻿{
   "ConnectionString": "Server=localhost;Database=VoloDocs;Trusted_Connection=True;MultipleActiveResultSets=true",
-  "LogoUrl": "/assets/images/Logo.png"
+  "LogoUrl": "/assets/images/Logo.png",
+
+  "ElasticSearch": {
+    "Url": "http://172.25.134.11:9200"
```
Comment (yekalkan) [inline modules/docs/app/VoloDocs.Web/appsettings.json]: We can change `http://172.25.134.11:9200` to `http://localhost:9200`

**Comment by yekalkan** [comment]
also, see https://github.com/abpframework/abp/issues/3009. We can think about it after this is merged.

**Comment by maliming** [comment]
@yekalkan pr has been updated.

**Comment by ebicoglu** [comment]
we need to add Elastic Search usage in https://docs.abp.io/en/abp/latest/Modules/Docs
Right now the project is not running without Elastic Search setup.

**Pull request merged**